dicom_standard/postprocess_merge_duplicate_nodes.py

```diff
@@ -0,0 +1,61 @@
+'''
+Merge nodes that have identical paths by adding conditional statments to the attribute descriptions.
+'''
+import re
+import sys
+from collections import Counter
+
+from bs4 import BeautifulSoup
+
+from dicom_standard import parse_lib as pl
+
+DUPLICATE_PATH_EXCEPTIONS = ['rt-segment-annotation:00700084']
+
+
+def get_description_text(raw_description):
+    description = BeautifulSoup(raw_description, "html.parser")
+    return description.getText()
+
+
+def add_conditional_to_description(node):
+    conditional = node.get('conditional')
+    assert conditional is not None, f'Duplicate attribute (path: {node["path"]}) has no conditional statement.'
+    conditional = re.sub(r'\.$', ':', conditional)
+    formatted_conditional = f'<p style="font-weight: bold">{conditional[0].upper()}{conditional[1:]}</p>'
+    node['description'] = formatted_conditional + node['description']
+
+
+def merge_duplicate_nodes(module_attr_list):
+    path_list = [d['path'] for d in module_attr_list]
+    duplicate_paths = [k for k, v in Counter(path_list).items() if v > 1]
+    path_to_node = {}
+    for node in module_attr_list:
+        path = node['path']
+        # Standard workaround: Catch inconsistency in Table C.36.8-1 where "Content Creator's Name" attribute
+        # appears twice in same hierarchy without a conditional
+        # http://dicom.nema.org/medical/dicom/2019c/output/chtml/part03/sect_C.36.8.html
+        if path not in DUPLICATE_PATH_EXCEPTIONS:
```

Comment:
It seems like this might remove "Content Creator's Name" entirely, right? Wouldn't we want to keep at least one of the attributes?